src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -308,6 +312,18 @@ static void upgradePreviewStyleAllowMarkdown(JabRefPreferences prefs) {
         prefs.setPreviewStyle(migratedStyle);
     }
 
+    static void changeColumnVariableNamesFor51(JabRefPreferences preferences) {
+        // The variable names have to be hardcoded, because they have changed between 5.0 and 5.1
```

Comment:
For me, it is OK to have local string constants. Externalizing them feels over-engineered.